boards/arm/stm32f4_disco/stm32f4_disco.dts

```diff
@@ -48,7 +48,10 @@
 	};
 
 	aliases {
-		led0 = &green_led_4;
+		led0 = &orange_led_3;
```

Comment:
Why are you modifiying led0?

Reply:
I wanted to keep the naming scheme consistent with other stm32 boards, where led0 always starts with the lowest ID.